kidkode/src/main/webapp/script.js

```diff
@@ -11,79 +11,73 @@ function getOrDefault(key, map) {
   }
 }
 
-
 /*
- * BUILD QUIZ
  * This function takes each question and their options & builds the HTML
  * elements of the quiz.
  */
-function buildQuiz() {
-  // variable to store the HTML output
-  const output = [];
-
-  // for each question...
-  myQuestions.forEach((currentQuestion, questionNumber) => {
-    // variable to store the list of possible answers
-    const answers = [];
-
-    // and for each available answer...
-    for (var option in currentQuestion.answers) {
-      // ...add an HTML radio button
+function buildQuiz(testQuestions) {
+  const output = []; // variable to store the HTML output
+
+  // for each set of questions, interate through each step and its items
+  var stepNumber = 0;
```

Comment:
Super nit-picky: do you think that this variable stepNumber might be easier to understand if it were named something like question number? Maybe it's because I wrote the JSON, but at first I was confused because the "step numbers" in the son don't match up what the actual number of questions there are. I could be interpreting the code wrong but it seems like this variable is just counting  the number of questions displayed on the page, right?

Reply:
Ah yes, good point! Yeah, `stepNumber` counts the number of questions on the page, so changing the name would make more sense.